app/models/congress_member.rb

```diff
@@ -201,12 +210,29 @@ def fill_out_form_with_capybara f={}, driver
         when "fill_in"
           if a.value.starts_with?("$")
             if a.value == "$CAPTCHA_SOLUTION"
-              location = CAPTCHA_LOCATIONS.keys.include?(bioguide_id) ? CAPTCHA_LOCATIONS[bioguide_id] : session.driver.evaluate_script('document.querySelector("' + a.captcha_selector.gsub('"', '\"') + '").getBoundingClientRect();')
-
-              url = self.class::save_captcha_and_store_poltergeist session, location["left"], location["top"], location["width"], location["height"]
+              if a.options and a.options["google_recaptcha"]
+                begin
+                  url = self.class::save_google_recaptcha_and_store_poltergeist(session,a.captcha_selector)
+                  captcha_value = yield url
+                  session.within_frame(recaptcha_frame_index(session)) do
+                    for i in captcha_value.split(",")
+                      session.execute_script("document.querySelector('.fbc-imageselect-checkbox-#{i}').checked=true")
+                    end
+                    sleep 0.5
+                    session.find(".fbc-button-verify input").trigger('click')
+                    @recaptcha_value = session.find("textarea").value
```

Comment:
We don't really use instance variables anywhere else, maybe we can just declare this nil above as a local var?